File: pinyougou-user-web/src/main/webapp/js/service/userService.js
app.service("userService",function ($http) {
    //点击获取验证码
    this.getValidCode=(phone)=>{
        return $http.get("./user/getValidCode.do?phone="+phone)
    }

    //注册
    this.add=(user,validCode)=>{
        return $http.post("./user/add.do?validCode="+validCode,user);
    }
})

File: pinyougou-user-interface/src/main/java/com/pinyougou/user/service/UserService.java
package com.pinyougou.user.service;

import com.pinyougou.pojo.TbUser;

/**
 * Created by WF on 2019/9/3 10:28
 */
public interface UserService {
    void sendValidCode(String phone);

    boolean checkValidCode(String phone, String validCode);

    void add(TbUser user);
}
